data/mods/MindOverMatter/PowerDescriptionSpoilers.md

```diff
@@ -1037,6 +1037,15 @@ Powers causing telepathic damage have a 5% chance to down the target, a 33% chan
 *Effects*: Travel a short distance through the Nether and re-emerge nearby, allowing the psion to bypass a door or travel from the ground floor to the second floor.<br />
 *Prerequisites*: Blink 6<br />
 
+## Phasing (C)
```

Comment:
Since there's already a power called Phase, is there a more distinct name that would work so they aren't confused for each other? Maybe like "Ephemeral Walk" or something similar?

Reply:
Switched everything over to Ephemeral Walk 👍